packages/components/pie-assistive-text/src/assistive-text.scss

```diff
@@ -13,6 +13,7 @@
     gap: var(--dt-spacing-b);
     display: flex;
     align-items: flex-start;
+    overflow-wrap: anywhere; // Break word will not have any effect inside a flex container
```

Comment:
As this is a relatively recent feature, does it just mean in older browsers (like iOS Safari 14) the text will just not wrap and look a little funky?

Reply:
I believe so yes. I would just add that this is not likely to happen in practice, I just couldn't ignore it 😂

Reply:
<img width="452" alt="Screenshot 2024-04-19 at 10 40 08" src="https://github.com/justeattakeaway/pie/assets/16766185/9cfbceb7-a0e1-45eb-ac79-d88d9c68cafb">

@ashleynolan this is on an iphone 12 running safari 14. With normal sentences it seems to behave fine. The css I added forces breaks if they just have 1 long continuous word as the assistive text.

In that case, on safari 14 it would look like this:
<img width="425" alt="Screenshot 2024-04-19 at 10 44 39" src="https://github.com/justeattakeaway/pie/assets/16766185/5ceba0f6-0ad6-4860-924d-d0666415da27">

Reply:
Yeh, that's fine. Pretty outlier case that will likely never happen 👍